### PR TITLE
Update description of `func` directive

### PR DESCRIPTION
**Pull request by jugmac00**
**Describe your changes**
I changed the description of the `func` directive.

I deduced this from reading the examples, not the source code - so take this with a grain of salt.

**Additional context**
I came here when trying to figure out how to proceed with https://github.com/tox-dev/tox/issues/2004

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Within the reStructuredText files use the `sphinx_argparse_cli` directive that t
 | Name        | Description                                                                                                                   |
 | ----------- | ----------------------------------------------------------------------------------------------------------------------------- |
 | module      | the module path to where the parser is defined                                                                                |
-| func        | the module path to where the parser is defined                                                                                |
+| func        | the function name of the parser                                                                                               |
```
Comment (gaborbernat) [inline README.md:37]: the name of the function that once called with no arguments constructs the parser

**Review by gaborbernat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jugmac00) [reply README.md:37]:
```suggestion
| func        | the name of the function that once called with no arguments constructs the parser                                             |
```

Right! Thanks, that makes sense.

Do you know of GitHub suggestions?
https://til.hashrocket.com/posts/szgdi4at59-github-insert-a-suggestion-

Comment (gaborbernat) [reply README.md]: Yes. I just don't like to use it 😆 too much hassle using it :-D

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**